apps/barista/src/styles/items/_color.scss

```diff
@@ -0,0 +1,147 @@
+.ba-color {
+  display: inline-block;
+  vertical-align: top;
+  min-width: 10rem;
+  width: 10rem;
+  margin-bottom: 1.5rem;
+
+  /* Color width if used without a colorgroup */
+  @media (min-width: $screen-lg) {
+    width: 24%; /* 4 colors per row */
+  }
+
+  @media (min-width: $screen-xl) {
+    width: 19%; /* 5 colors per row */
+  }
+
+  @media (min-width: $screen-xxl) {
+    width: 16%; /* 6 colors per row */
+  }
+
+  @media (min-width: $screen-xxxl) {
+    width: 14%; /* 7 colors per row */
+  }
+}
+
+.ba-color__block {
```

Comment:
Could we remove BEM-syntax?

Reply:
I did not change the BEM-syntax because I copied the HTML from the different color pages and they contained this syntax. It is only a temporary solution and BEM will be removed as soon as a color component is implemented.